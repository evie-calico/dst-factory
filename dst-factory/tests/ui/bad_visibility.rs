use dst_factory::make_dst_factory;

#[make_dst_factory(create, XYZ)]
struct BadVisibility {
    id: i32,
    data: str,
}

fn main() {
}
