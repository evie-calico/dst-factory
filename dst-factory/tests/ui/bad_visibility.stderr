error: Unexpected input
 --> tests/ui/bad_visibility.rs:3:28
  |
3 | #[make_dst_factory(create, XYZ)]
  |                            ^^^
